src/FileInfo.cpp

```diff
@@ -57,8 +57,6 @@ FileInfo GetFileInfo(const char *path)
     if ((stbuf.st_mode & S_IFMT) == S_IFLNK)
     {
         flags |= FileInfo::kFlagSymlink;
-        if (0 != stat(path, &stbuf))
```

Comment:
should this S_IFLNK check move down lower with the rest of the S_IFMT checks?

Reply:
Moved. This now also applies to windows (IIRC Vista's NTFS onwards supports symlinks). So another case that I will try to test.